src/Umbraco.Web/PublishedCache/NuCache/DataSource/MsgPackContentNestedDataSerializer.cs

```diff
@@ -38,7 +38,8 @@ public MsgPackContentNestedDataSerializer(IPropertyCacheCompression propertyOpti
 
             _options = defaultOptions
                 .WithResolver(resolver)
-                .WithCompression(MessagePackCompression.Lz4BlockArray);            
+                .WithCompression(MessagePackCompression.Lz4BlockArray)
+                .WithSecurity(MessagePackSecurity.UntrustedData);            
```

Comment:
what does that one do?

Reply:
Helps prevent serialising types that are not safe. https://github.com/neuecc/MessagePack-CSharp#security. It's really just a start. It should limit to just primitives and expected types.